src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int baseLength;
+    private int height;
+
+    public RightTriangle(String color, int height, int bothside) {
+        super(color);
+        this.height = height;
+        this.baseLength = bothside;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle");
+        System.out.println("Color: " + getColor());
+        System.out.println("Both side length: " + baseLength);
```

Comment:
The message 'Both side length' may be misleading as it implies both sides are of equal length, which is not necessarily the case for a right triangle. Consider changing the message to 'Base length' to match the variable name and provide clarity.